README.md

```
@@ -1,78 +1,421 @@
+
+このプロジェクトでは、[openapi-ts](https://github.com/hey-api/openapi-ts)を使用してAPIの型定義とZodスキーマを自動生成しています。
+
+### 概要
+
+- APIスキーマから TypeScript の型定義と Zod スキーマを自動生成
+- 生成されたファイルは `shared/types/api/` ディレクトリに配置
+- フロントエンドでの API レスポンスの型安全性とランタイムバリデーションを提供
+
+### 生成されるファイル
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)